pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -389,8 +390,28 @@ func (i *KumaInjector) sidecarEnvVars(mesh string, podAnnotations map[string]str
 }
 
 func (i *KumaInjector) NewVolumeMounts(pod *kube_core.Pod) []kube_core.VolumeMount {
+	volumeMounts := []kube_core.VolumeMount{}
 	if tokenVolumeMount := i.FindServiceAccountToken(pod); tokenVolumeMount != nil {
-		return []kube_core.VolumeMount{*tokenVolumeMount}
+		volumeMounts = append(volumeMounts, *tokenVolumeMount)
+	}
+	if datadogVolumeMount := i.FindDatadogMountPath(pod); datadogVolumeMount != nil {
+		volumeMounts = append(volumeMounts, *datadogVolumeMount)
+	}
+
+	if len(volumeMounts) < 1 {
+		return nil
+	}
+
+	return volumeMounts
+}
+
+func (i *KumaInjector) FindDatadogMountPath(pod *kube_core.Pod) *kube_core.VolumeMount {
```

Comment:
Is it possible to add a test case in `injector_test.go` to cover this case with datadog mount?